test/logger.test.js

```diff
@@ -182,23 +184,30 @@ describe('logger', () => {
 
             logger._createBulk.restore();
             logger.close();
+        });
+
+        it('valid nano-sec timestamp', (done) => {
+            var mockTime = 0.123456;
+            var expectedLogTime = '000123456';
 
-            // testing with nano seconds
             logger = createLogger({
                 bufferSize: 1,
                 callback: done,
                 addTimestampWithNanoSecs: true,
             });
             sinon.spy(logger, '_createBulk');
-
+            hrtimemock(mockTime);            
+            process.hrtime();
```

Comment:
Why do you need this?
Remove if unnecessary

Reply:
That's the nanosec-timestamp mock - it sets the _addtimestamp function to get a specific nanosec time.